packages/dd-trace/src/appsec/remote_config/capabilities.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
   APM_TRACING_LOGS_INJECTION: 1n << 13n,
   APM_TRACING_HTTP_HEADER_TAGS: 1n << 14n,
   APM_TRACING_CUSTOM_TAGS: 1n << 15n,
-  APM_TRACING_ENABLED: 1n << 19n
+  APM_TRACING_ENABLED: 1n << 19n,
+  APM_TRACING_SAMPLE_RULES: 1n << 29n
```

Comment:
you're adding a new capability but you're never using it ?

Reply:
good catch